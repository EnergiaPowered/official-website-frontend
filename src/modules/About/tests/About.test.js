import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import About from '../components/page/index';

import { create } from "react-test-renderer";

// because it's static component and doesn't change often
it('matches snapshot', () => {
  const component = create(
    <Router>
      <About />
    </Router>
  );

  expect(component.toJSON()).toMatchSnapshot()
});
