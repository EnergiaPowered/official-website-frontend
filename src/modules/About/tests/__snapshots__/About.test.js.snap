// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="page-component"
  id="About"
  style={
    Object {
      "backgroundImage": "url(About-header.png)",
    }
  }
>
  <nav
    className="navbar fixed-top navbar-expand-sm "
    data-testid="navbar"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand "
        href="/"
        onClick={[Function]}
      >
        <img
          alt="Energia's Logo"
          className="logo-img"
          src="logo.png"
          title="logo of the team"
        />
      </a>
      <button
        aria-controls="navbarNav"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-testid="toggler"
        onClick={[Function]}
        type="button"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-bars fa-w-14 navbar-toggler-icon"
          data-icon="bars"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </button>
      <div
        className="navbar-collapse"
        data-testid="side-menu"
        id="navbarNav"
        onClick={[Function]}
        style={
          Object {
            "right": "-90%",
          }
        }
      >
        <button
          className="menu-close"
          data-testid="closer"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-times fa-w-11 "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 352 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
        <ul
          className="navbar-nav ml-auto"
          data-testid="menu-list"
          onClick={[Function]}
        >
          <li
            className="nav-item"
          >
            <a
              aria-current="page"
              className="nav-link active"
              data-testid="navlinks"
              href="/"
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/about-us"
              onClick={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/blogs"
              onClick={[Function]}
            >
              Blogs
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/events"
              onClick={[Function]}
            >
              Events
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/crew"
              onClick={[Function]}
            >
              Crew
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/contact-us"
              onClick={[Function]}
            >
              Contact us
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/signup"
              onClick={[Function]}
            >
              Sign Up
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-current={null}
              className="nav-link"
              data-testid="navlinks"
              href="/login"
              onClick={[Function]}
            >
              Log In
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <section
    className="header-section"
    id="header"
  >
    <header
      className="container"
    >
      <h1
        className="header-title"
      >
         Know More About Us 
      </h1>
    </header>
  </section>
  <main>
    <div
      className="container"
    >
      <section
        className="bg-section about-vision component-font"
      >
        <h2
          className="section-title"
        >
          Our Vision
        </h2>
        <p
          data-testid="vision-text"
        >
           
          Perparing calibers by developing students in both personal and career levels to make them qualified for the market needs.
           
        </p>
      </section>
      <hr />
      <section
        className="bg-section about-mission component-font"
      >
        <h2
          className="section-title"
        >
          Our Mission
        </h2>
        <p>
          Energia Powered attempts to deliver values and achieve its vision through different projects:
        </p>
        <ul>
          <li>
             Conference 
          </li>
          <li>
             Training Sessions 
          </li>
          <li>
             Events 
          </li>
          <li>
             Field Trips 
          </li>
        </ul>
      </section>
      <hr />
      <section
        className="bg-section about-structure"
      >
        <h2
          className="section-title"
        >
          Our Structure
        </h2>
        <div
          className="d-flex justify-content-center"
        >
          <div
            className="EnlargedImage"
          >
            <div
              className="EnlargedImage__container"
              style={
                Object {
                  "display": "inline-block",
                  "position": "relative",
                }
              }
            >
              <img
                alt="Our team's structure"
                className="EnlargedImage__Image"
                data-testid="structure-image"
                data-teststate={false}
                onClick={[Function]}
                src="Structure.jpg"
                style={
                  Object {
                    "cursor": "zoom-in",
                    "height": "auto",
                    "maxWidth": "100%",
                    "opacity": 1,
                  }
                }
                title="Energia Powered'20 board"
              />
            </div>
          </div>
        </div>
      </section>
    </div>
  </main>
  <footer
    id="Footer"
  >
    <ul
      className="text-center social-links row list-unstyled"
    >
      <li
        className="social-links-item col-xs-2"
      >
        <a
          href="https://www.facebook.com/Energia.Powered/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-facebook fa-w-16 "
            data-icon="facebook"
            data-prefix="fab"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </li>
      <li
        className="social-links-item col-xs-2"
      >
        <a
          href="https://www.youtube.com/channel/UCxNgZOJJx9LzqWLhCEu8Djw"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-youtube fa-w-18 "
            data-icon="youtube"
            data-prefix="fab"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </li>
      <li
        className="social-links-item col-xs-2"
      >
        <a
          href="https://www.instagram.com/energia.powered/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-instagram fa-w-14 "
            data-icon="instagram"
            data-prefix="fab"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </li>
      <li
        className="social-links-item col-xs-2"
      >
        <a
          href="https://twitter.com/Energia_Powered"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-twitter fa-w-16 "
            data-icon="twitter"
            data-prefix="fab"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </li>
      <li
        className="social-links-item col-xs-2"
      >
        <a
          href="https://www.linkedin.com/company/energiapowered/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-linkedin fa-w-14 "
            data-icon="linkedin"
            data-prefix="fab"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </li>
    </ul>
  </footer>
</div>
`;
